src/mixxx.cpp

```diff
@@ -171,6 +171,13 @@ MixxxMainWindow::MixxxMainWindow(QApplication* pApp, const CmdlineArgs& args)
     m_runtime_timer.start();
     mixxx::Time::start();
 
+#ifdef __APPLE__
+    Sandbox::checkSandboxed();
```

Comment:
Please add a comment about the restrictions that have been violated before, i.e. **after** creating the application.